### ISSUE TITLE
Ignore slice-file for ReferencedUnitNotFound

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/systemdlint#34]
otherwise this false positive will be generated
```
rootfs/etc/systemd/system/NetworkManager.service.d/service.NetworkManager.conf :2:1 - The Unit 'system.slice' referenced was not found in filesystem - [systemdlint.systemdlint.ReferencedUnitNotFound]
```
this applies to all *.slice files, as they are created at runtime
